### DISCUSSION TITLE
Razor support

### DISCUSSION THREAD

**Discussion by Ashijo** [super-linter/super-linter#4791] [Q&A] [answered]
Any plan for supporting Razor in the near future ?

**Comment by zkoppert** [comment, marked as answer]
Not that I am aware of. Do you have a link to the Razor linter (I'm assuming its a linter, I am not actually familiar with it)? Is this something you would like to implement?

**Reply by Ashijo** [reply]
I am looking for one, but no luck yet, hoped you maybe had an idea.
Dotnet cli side it doesn't seem to be anytime soon neither for coderush. Maybe jetbrain will release something for the resharper cli or maybe blazor is just not popular enough for anyone to bother making one yet ^^,
